Swissdox/Swissdox.py

```diff
@@ -38,15 +34,23 @@ def __init__(self,
             self.default_folders = ["raw-data", "intermediate-data"]
 
         self.DEFAULT_SCHEMA_VERSION = "1.2"
-        self.SCHEMA_CONFIG_PATH = os.path.normpath(os.path.join(Path(__file__).parent.resolve(), "schemas/available-schemas.yaml"))
+        self.SCHEMA_CONFIG_PATH = os.path.normpath(
+            os.path.join(Path(__file__).parent.resolve(), "schemas/available-schemas.yaml"))
 
     def initialize_dataset(self, data_root_folder, validate_query_file=True, query_string=None):
+        """
+        Initialize dataset from the given data root directory.
+        :param data_root_folder:
```

Comment:
Explaining all parameters would be helpful. just a couple key words / one sentence per parameter

Reply:
plus their type as well. THen when one hovers over the function the docstrings are displayed and everything is clear

Reply:
Thank you, I have adjusted it in most cases